src/Tools/IdeCoreBenchmarks/CloudCache/SolutionServiceMock.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+using Microsoft.VisualStudio.RpcContracts.Solution;
+using Microsoft.VisualStudio.Threading;
+
+#pragma warning disable CS0067 // events that are never used
+
+namespace CloudCache
+{
+    internal class SolutionServiceMock : ISolutionService
```

Comment:
These are stubs, not mocks.

```suggestion
    internal class StubSolutionService : ISolutionService
```

See https://github.com/dotnet/roslyn/pull/34117#discussion_r274044079 for prior discussion on this.